test/integration/test-amp-analytics.js

```diff
@@ -17,9 +17,7 @@
 import {BrowserController, RequestBank} from '../../testing/test-helper';
```

Comment:
The `should trigger 1s after amp-analytics starts` test in this file is flaky because it tries to assert that 999 > 1000. Probably because the timer for setInterval isn't always accurate.

Reply:
good observation. changed 1000 to 950 to give some tolerance